CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## UNRELEASED
 
+### Changed
+- ([#91](https://github.com/demos-europe/demosplan-ui/pull/91)) Pass routes as props ([@ahmad-demos](https://github.com/ahmad-demos))
```

Comment:
this is BREAKING.

And we should mention That `Routing.generate` is not used internally anymore. (in most the places)

Reply:
If I understood correctly, breaking changes are not mentioned as such until version 1?
Also, I'm not sure we need to mention internal changes?

Reply:
Ok. I thought we should mention this, because its not internal since You have to pass (other) props/values in the components. 

Its just possible to make braeaking changes without bumping a major version (but with telling in the chagnelog). That was, what was I thought.

Reply:
but I am not sure at all